src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -306,7 +306,7 @@ public Conference createConference(
         // The method Videobridge.getConferenceCountString() should better
         // be executed outside synchronized blocks in order to reduce the
         // risks of causing deadlocks.
-        if (logger.isInfoEnabled())
+        if (logger.isInfoEnabled() && enableLogging)
```

Comment:
are we sure we don't want to keep these for health checks?  since the health check confs already disable logging, i worry there won't be much evidence in the logs that they're happening at all, which may be useful?

Reply:
Maybe you're right, I'll remove this.